src/OrchardCore.Modules/OrchardCore.Security/Extensions/OrchardCoreBuilderExtensions.cs

```diff
@@ -14,6 +14,9 @@ public static OrchardCoreBuilder ConfigureSecuritySettings(this OrchardCoreBuild
 
                 tenantServices.PostConfigure<SecuritySettings>(settings =>
                 {
+                    settings.ContentSecurityPolicy.Clear();
```

Comment:
Similar to what I did before when I reset the arrays ;)

Reply:
Yes you're right ;)

Reply:
Let me did a quick look again to finalize the PR, everything seems ok